hlapipeline/alignimages.py

```diff
@@ -273,20 +309,54 @@ def perform_align(input_list, archive=False, clobber=False, update_hdr_wcs=False
                         print("Not enough cross matches found in any catalog - no processing done.")
                         return(1)
                 elif max_rms_val > MAX_FIT_RMS:
-                    if catalogIndex < numCatalogs-1:
-                        print("Fit RMS value = {}mas greater than the maximum threshold value {}.".format(item.meta['tweakwcs_info']['FIT_RMS'].value,MAX_FIT_RMS))
-                        print("Try again with the next catalog")
-                        catalogIndex += 1
-                        retry_fit = True
-                        break
+                    if catalogIndex <= numCatalogs-1:
+                        print("Fit RMS value = {}mas greater than the maximum threshold value {}.".format(item.meta['tweakwcs_info']['FIT_RMS'].value, MAX_FIT_RMS))
+                        if iter_ctr <= MAX_ITERATIONS:
+                            #new_tolerance = math.ceil(radial_shift)
```

Comment:
I *think* the logic here on the number of iterations does not need to be inside of the catalogIndex check.  You should just be able to change the tolerance and try the TPMatch again.